### DISCUSSION TITLE
U_RHEL_8_STIG_V1R12 STIG Benchmark Import Errors

### DISCUSSION THREAD

**Discussion by Vesfoldy** [NUWCDIVNPT/stig-manager#1168] [General]
I downloaded the U_RHEL_8_STIG_V1R12 benchmark and get an "unexpected character at line 1 column 1 of the JSON data" error when trying to import it.  Can anyone help me get this benchmark imported?
[U_RHEL_8_V1R12_STIG.zip](https://github.com/NUWCDIVNPT/stig-manager/files/13746435/U_RHEL_8_V1R12_STIG.zip)

**Comment by cd-rite** [comment]
Hi @Vesfoldy In most cases, that is the result of a proxy that's restricting the max body size on requests. That RHEL_8 STIG is getting really big! The comments on this issue should help you fix your issue:
https://github.com/NUWCDIVNPT/stig-manager/issues/1148#issuecomment-1810398353

In that case with nginx, they needed to set client_max_body_size to 2M, since the default is pretty small.

**Reply by Vesfoldy** [reply]
@cd-rite that did the trick. Thanks!
